Library/Homebrew/config.rb

```diff
@@ -39,4 +39,4 @@
 HOMEBREW_LIBRARY_PATH = Pathname.new(__FILE__).realpath.parent
 
 # Load path used by standalone scripts to access the Homebrew code base
-HOMEBREW_LOAD_PATH = HOMEBREW_LIBRARY_PATH
+HOMEBREW_LOAD_PATH = HOMEBREW_LIBRARY_PATH.to_s
```

Comment:
Sorry my bad, I misread the code to `HOMEBREW_LIBRARY_PATH = HOMEBREW_LIBRARY_PATH.to_s`. Please ignore my last comment